src/aap_eda/services/activation/manager.py

```diff
@@ -1050,6 +1073,23 @@ def _create_activation_instance(self):
             "status": ActivationStatus.STARTING,
             "git_hash": git_hash,
         }
+        if settings.LOCAL_QUEUE_NAME:
+            args["node"], _ = models.Node.objects.get_or_create(
+                name=settings.LOCAL_QUEUE_NAME
+            )
+            monitor_events_job_id = settings.LOCAL_QUEUE_NAME
+            monitor_events_queue = settings.LOCAL_QUEUE_NAME
+        else:
+            monitor_events_job_id = "activation"
+            monitor_events_queue = "activation"
+
+        # Start the Monitor Engine events once
+        if not ActivationManager.started_monitor_events:
+            ActivationManager.started_monitor_events = True
+            start_monitor_engine_events_job(
+                monitor_events_queue, monitor_events_job_id
```

Comment:
`monitor_events_job_id` should not be of the form "{queue}-{activation-id}"? otherwise the unique_enqueue will discard it

Reply:
@Alex-Izquierdo its not a periodic job, its a dedicated activation worker listening to events from the container engine. It's started by the Manager once with a unique id and it will block a single activation worker to listening the events. Doing busy loop anywhere is not a good design if we can handle events coming back from the engine we can completely eliminate the busy loop approach. If there are 3 activation workers on a node, 1 worker is dedcated to listening for events from the engine and the 2 workers do the start/stop/monitoring